bolt/lib/Core/Exceptions.cpp

```diff
@@ -632,8 +643,13 @@ bool CFIReaderWriter::fillCFIInfoFor(BinaryFunction &Function) const {
       // DW_CFA_GNU_window_save and DW_CFA_GNU_NegateRAState just use the same
```

Comment:
I think the author made a typo here. Could you fix that in your patch as well?
(I'm asking this as I have related suggestions elsewhere in my review)

```suggestion
      // DW_CFA_GNU_window_save and DW_CFA_AARCH64_negate_ra_state just use the same
```